### PR TITLE
Move public router tf module to ansible role

### PR DESCRIPTION
**Pull request by Polina-Gubina**
Move public router tf module to ansible role

### CODE REVIEW

**Review by anton-sidelnikov — DISMISSED** [review]

File: roles/public_router/vars/main.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+prefix: "{{ scenario_name }}"
```
Comment (outcatcher) [inline roles/public_router/vars/main.yaml]: Redundant variable

File: roles/public_router/tasks/main.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+- name: Build infra
```
Comment (outcatcher) [inline roles/public_router/tasks/main.yaml]: Maybe better name it `Build Router` and `Destroy Router``

**Review by outcatcher — CHANGES_REQUESTED** [review]
Please add readme with variable definition

File: roles/public_router/defaults/main.yaml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+subnet_cidr: "192.168.0.0/24"
+scenario_name: "public_router"
```
Comment (outcatcher) [inline roles/public_router/defaults/main.yaml]: Please add final new line

**Review by outcatcher — CHANGES_REQUESTED** [review]

**Review by outcatcher — APPROVED** [review]

**Pull request merged**